compiler+runtime/src/jank/clojure/core.jank

```diff
@@ -5101,22 +5101,10 @@
     ;;    (. latch (await  timeout-ms (. java.util.concurrent.TimeUnit MILLISECONDS)))))
   (throw "TODO: port await-for"))
 
-(defmacro import
-  "import-list => (package-symbol class-name-symbols*)
-
-  For each name in class-name-symbols, adds a mapping from name to the
-  class named by package.name to the current namespace. Use :import in the ns
-  macro in preference to calling this directly."
-  [& import-symbols-or-lists]
-  (let [specs (map #(if (and (seq? %) (= 'quote (first %))) (second %) %)
-                   import-symbols-or-lists)]
-    `(do ~@(map #(list 'clojure.core/import* %)
-                (reduce (fn [v spec]
-                          (if (symbol? spec)
-                            (conj v (name spec))
-                            (let [p (first spec) cs (rest spec)]
-                              (into v (map #(str p "." %) cs)))))
-                        [] specs)))))
+(defn import
+  "import is not implemented for jank, but a var is still bound to its symbol for portability. import always throws an exception"
```

Comment:
I could also add the suggestion about the workaround in case folks are using `(doc import)` in the repl and looking to understand how to use interop in jank